packages/laconia-event/src/apigateway/ApiGatewayWebsocketEvent.js

```diff
@@ -0,0 +1,18 @@
+const parseWebsocket = require("./parseWebsocket");
+const ApiGatewayWebsocketContext = require("./ApiGatewayWebsocketContext");
+
+module.exports = class ApiGatewayWebsocketEvent {
+  constructor(body) {
+    this.body = body;
+  }
+
+  static fromRaw(event) {
+    const apiGatewayWebsocketEvent = new ApiGatewayWebsocketEvent(
+      parseWebsocket(event)
+    );
+    apiGatewayWebsocketEvent.context = new ApiGatewayWebsocketContext(
```

Comment:
What's your thought on the variable name? Should we keep it to `requestContext` or change it to `context`? I'm always torn apart on this one..

Reply:
I liked `context` more, I had it the other way before